test/SILOptimizer/package-cmo-resilient-mode.swift

```diff
@@ -188,6 +189,20 @@ public func mainPubArgRetInlinable(_ arg: PubKlass) -> PubStruct {
   return PubStruct(arg.data)
 }
 
+// CHECK-OPAQUE: sil @$s4Main023testPackageInSerializedC4FuncyyF : $@convention(thin) () -> ()
```

Comment:
Thanks for the fix! Could you create a new test file for this, like package-cmo-opaque-type? This file already contains too much and is getting harder to track new cases.